test/EthLightClient.js

```diff
@@ -0,0 +1,150 @@
+const { expect } = require("chai");
+const { ethers, upgrades } = require("hardhat");
+const block1 = require('./proof1.json');
+const block2 = require('./proof2.json');
+
+function hexToBytes(hex) {
+    for (var bytes = [], c = 0; c < hex.length; c += 2)
+        bytes.push(parseInt(hex.substr(c, 2), 16));
+    return bytes;
+}
+
+const DEBUG = true;
+
+function debug(msg){
+    if(DEBUG) console.log(msg);
+}
+
+
+describe("Ethereum Light Client", function () {
```

Comment:
Lots of variables used without declaration in this file?

Reply:
Fixing this

Reply:
Fixed